studio/lib/helpers.ts

```diff
@@ -32,14 +33,8 @@ export const prettifyJSON = (minifiedJSON: string) => {
   }
 }
 
-// https://stackoverflow.com/a/2117523
-// a "good enough" unique ID that typescript is happy with, and doesn't have external dependencies
 export const uuidv4 = () => {
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
-    var r = (Math.random() * 16) | 0,
-      v = c == 'x' ? r : (r & 0x3) | 0x8
-    return v.toString(16)
-  })
```

Comment:
is `nanoid` worth exploring here? https://www.npmjs.com/package/nanoid
mainly because its smaller and faster than uuid
cc @phamhieu (did this package cause problems for us before? i vaguely recall)

Reply:
Unfortunately, we need to use uuids here because they get saved into the database (unless we want to change the column type). It's less than 500 bytes bigger than `nanoid`, so size shouldn't be too much of a concern.
Maybe the problem you had with it was when they did a major version upgrade and broke everyone's installs? I'm interested to hear from @phamhieu here too!

Reply:
okay can do - i don't think this should be an issue
cc @supabase/frontend just fyi for the team